src/Controls/tests/TestCases.HostApp/Issues/Issue28334.xaml.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 28334, "[Windows] EmptyViewTemplate not displayed when ItemsSource is set to Null", PlatformAffected.UWP)]
+public partial class Issue28334 : ContentPage
+{
+	private Issue28334CollectionViewViewModel _viewModel;
+	public Issue28334()
+	{
+		InitializeComponent();
+		this.BindingContext = _viewModel = new Issue28334CollectionViewViewModel();
+	}
+	private void OnItemSourceButtonClick(object sender, EventArgs e)
+	{
+		_viewModel.ItemsSource = null;
```

Comment:
Could include also a test (and Button here in the sample) to re-fill also the ItemSource? The scenario will be like, set the ItemsSource to null, verify the EmptyView and then, restore the data to verify that the EmptyView is removed and renders the items correctly.

Reply:
I have included a test to refill the `ItemsSource` after the EmptyViewTemplate is displayed. Please let me know if you have any other concerns. @jsuarezruiz